### ISSUE TITLE
landmark loss decreased slowly

### ISSUE THREAD

**Issue by anewusername77** [abdallahdib/DeepNextFace#3]
Hey there, thanks for your impressive work!
I tried Nextface first and now implemented DeepNextFace  by myself, using default resnet152 in pytorch(pretrained on ImageNet), but during step 1, landmark loss decreased in a quite slow way,  ended up with about 40000.  I think it may related to slow fitting of focal and camera position params.
Dosen't find more details about the training strategy in your paper including appendix, can you please help?
